src/plays/star-rating/styles.css

```diff
@@ -0,0 +1,30 @@
+/* enter stlyes here */
+.play-details .play-details-body {
```

Comment:
lets not use the generic name is CSS. Rather you can use specific play related names.
like
.star-rating-wrapper
.star-rating-container

etc

Reply:
I noticed the scrolls is from the `.play-details`

Reply:
yes, scroll is from play-details. I was mentioning about the class name in [src/plays/star-rating/styles.css](https://github.com/reactplay/react-play/pull/513/files/48d7820bbb007c95e3c040e07b967007373cc524#diff-207ad5ab6ede404afc855c1e809c90b4d86a9d5a9759a6d6f50293d477a1810f).. Do not try to override and lets use unique play specific names.

Reply:
Okay